cabal.GHC-9.2.4.config

```diff
@@ -0,0 +1,299 @@
+active-repositories: hackage.haskell.org:merge
```

Comment:
Now that we have a freeze file for 9.2.4, we should be able to get rid of the ones for 9.2.2 and 9.2.3, no?

Reply:
Generally, yes.  The caveat would be people still trying to build with 9.2.2 and 9.2.3 (similar to the previous comment).  My general thinking is that 9.2 roughly represents our "current" target (9.4 is the leading edge, and 8.10 is the trailing edge ... we can support older than 8.10 if convenient, but we aren't committing to it), and I usually look to contract the support range when things move from current to trailing.